chunky/src/java/se/llbit/math/Grid.java

```diff
@@ -141,17 +145,26 @@ public EmitterPosition sampleEmitterPosition(int x, int y, int z, Random random)
     return emitterPositions.get(emitterIndex);
   }
 
+  private boolean isOOB(int coord)
```

Comment:
Maybe rename this to `isOutOfBounds`? And maybe `isOutOfBounds(int x, int y, int z)` just in case the grid gets non-cubic in the future?

Reply:
Making the grid non cubic is indeed the next step (but I was too lazy to do that today) so this change makes sense